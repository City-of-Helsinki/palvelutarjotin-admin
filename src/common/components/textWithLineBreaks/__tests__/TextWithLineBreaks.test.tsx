import { shallow } from 'enzyme';
import * as React from 'react';

import TextWithLineBreaks from '../TextWithLineBreaks';

it('TextWithLineBreaks matches snapshot', () => {
  const text = `Line 1
  Line 2`;
  const component = shallow(<TextWithLineBreaks as="div" text={text} />);
  expect(component.html()).toMatchSnapshot();
});
