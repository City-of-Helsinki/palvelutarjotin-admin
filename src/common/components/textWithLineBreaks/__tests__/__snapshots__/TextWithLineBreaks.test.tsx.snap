// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`TextWithLineBreaks matches snapshot 1`] = `
<div>
  Line 1
  <br />
    Line 2
</div>
`;
